### PR TITLE
Add next next next meeting

### PR DESCRIPTION
**Pull request by robpalme**

### CODE REVIEW

**Review by ctcpip — APPROVED** [review]

File: 2023/09.md [hunk 1]
```diff
@@ -195,3 +195,4 @@ When applicable, use these emoji as a prefix to the agenda item topic.
 |--------------------------|---------------------|----------------------|
 | 2023-11-27 to 2023-11-30 | *Remote: "SF"*      |                      |
 | 2023-02-06 to 2023-02-08 | San Diego, US       | ServiceNow           |
+| 2023-04-08 to 2023-04-11 | *Remote: "Baltimore"* |                    |
```
Comment (aciccarello) [inline 2023/09.md:198]: I assume this meeting (and the San Diego one) are supposed to be in 2024?

**Comment by ljharb** [comment]
What ecma member is Baltimore a standin for?

**Comment by robpalme** [comment]
Thanks for catching the typo! 

@ljharb We don't associate remote plenaries with hosts/members. The "location" exists to anchor to a timezone (east coast US) and permit levity.

**Comment by ljharb** [comment]
Ah, historically they represented where the meeting would have been held in person, and which member would have hosted.

**Pull request merged**